.github/workflows/correctness-linux.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
         spack install googletest
         spack install cppcheck
         spack install yaml-cpp
-        spack install trilinos@master~mpi~epetra
+        spack install trilinos@master~epetra
```

Comment:
I thought we could get away without putting `mpi` into CI -- I suppose it's a check for the code?

Reply:
Yes, this is just to check to ensure it continues to work.  The MacOS version still tests with a version of Trilinos built without MPI, so both cases are covered. 

In truth, even Linux CI is still using the old cache right now, but it will build a new Trilinos with MPI next time we rebuild it.